dotnet-client-libraries/OpenAIImageGeneration/Tests/Tests/ImageGeneratorControllerTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Moq;
+using OpenAiImageGenerator;
+using OpenAiImageGenerator.Controllers;
+using OpenAiImageGenerator.Services;
+using System.Net;
```

Comment:
I think these should be moved to the `GlobalUsings.cs` file, especially since they'd likely be present in every test class, if this project had multiple test classes...